cgo/lapack.go

```diff
@@ -737,19 +736,19 @@ func (impl Implementation) Dorgbr(vect lapack.DecompUpdate, m, n, k int, a []flo
 
 // Dorglq generates an m×n matrix Q with orthonormal rows defined by the
 // product of elementary reflectors as computed by Dgelqf.
-//  Q = H(0) * H(2) * ... * H(k-1)
-// Dorglq is the blocked version of dorgl2 that makes greater use of level-3 BLAS
+//  Q = H_k ... H_2 * H_1
```

Comment:
I think this should be zero indexed. I tried to be consistent with that in the documentation. It doesn't matter for the documentation of this function, but it does for others.